spec/components/post_revisor_spec.rb

```diff
@@ -213,6 +213,21 @@
         end
       end
 
+      context "invalid description without paragraphs" do
+        before do
+          subject.revise!(post.user, { raw: "# This is a title" })
+          category.reload
+        end
+
+        it "returns a error for the user" do
+          expect(post.errors.present?).to eq(true)
```

Comment:
We should test for the description of the error otherwise another error may cause this test to incorrectly pass.